src/config/args.hpp

```diff
@@ -145,6 +145,8 @@
 #define LBA_RECONSTRUCTION_BATCH_SIZE             1024
 
 #if defined (__powerpc64__)
+// POWER ABI requires more number of registers to be saved
+// in stack during context switch.
```

Comment:
@mundaym There are alloca() calls in glibc called by getifaddrs() which tries to
allocate 64KB memory in stack frame for powerpc. If I dont increase the stack
size, there are failures in test cases due to stack overflow checks in rethinkdb.
Hence increasing it for powerpc.